### ISSUE TITLE
Failing Playwright tests

### ISSUE THREAD

**Issue by pli888** [gigascience/gigadb-website#1653]
When `docker-compose run --rm playwright npm run test` is executed, there are 3 a11y Playwright tests failing in the `dev` environment:

```
 3 failed
    [chromium] › accessibility.spec.js:61:7 › Page / › should not have any color contrast issues ───
    [firefox] › accessibility.spec.js:61:7 › Page / › should not have any color contrast issues ────
    [webkit] › accessibility.spec.js:61:7 › Page / › should not have any color contrast issues ─────
  71 passed (43.4s)
```
